### DISCUSSION TITLE
Welcome to transloco Discussions!

### DISCUSSION THREAD

**Discussion by NetanelBasal** [ngneat/transloco#381] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Coly010** [comment]
# HI :wave: 

### I'm Colum, I'm a Software Engineer at This Dot Labs, working in the JavaScript space, mostly with Angular.

I'm not as active a contributor as I used to be as I have much less time these days, but I do try to help out when I can. 

I built the original MessageFormat Transpiler for Transloco which has been turned into an [Official Plugin](https://ngneat.github.io/transloco/docs/plugins/message-format) and is incredible for handling Gender and Pluralization in your Translations!

I've also written an article on using Transloco with Angular!
https://labs.thisdot.co/blog/internationalization-i18n-in-angular-with-transloco

**Comment by Dursunali07** [comment]
Merhaba millet biri bana anlatsın bu nedir ben burda ne yapabilirim burya nasıl geldim
